conf/base.config

```diff
@@ -27,25 +27,25 @@ process {
     withLabel:process_single {
         cpus   = { check_max( 1                  , 'cpus'    ) }
         memory = { check_max( 6.GB * task.attempt, 'memory'  ) }
-        time   = { check_max( 4.h  * task.attempt, 'time'    ) }
+        time   = { check_max( 8.h  * task.attempt, 'time'    ) }
```

Comment:
The idea of the `base.config` is that it is exactly the same across all pipelines. If a particular process needs more resources, then this can be specified in the `modules.config` file